### ISSUE TITLE
Widget gets clipped for no apparent reasons.

### ISSUE THREAD

**Issue by aj-Thapa** [flutter/flutter#149706]
### Steps to reproduce

1. Run the minimal code attached below.
2. Initially the DropdownMenu, in the first **input Row** widget, I've used appeared alright, but upon adding more **result Rows** below in the code, as you should find in the minimal code, the DropdownMenu started to show slightly clipped behind the margins. As a workaround to this problem, I added a sized box of sufficient width to push the side of the DropdownMenu within the viewable area, without clipping. Now, the widget looked good in this emulator, but when I ran the app on my Android 13 device, the problem arrived again, clipped DropdownMenu, despite SizedBox.
3. In my previous Github issue, this problem was marked solved and closed without the problem actually being solved. The solution applied there was to replace `Expanded` widget with `Flexible` widget, but it worked in only one of a few of my pages that use similar layout and codes. I updated my code where the fix does not get the problem solved but the question was not reopened despite a week of request. So, I'm creating this new issue. This is the link to my previous [issue #148667](https://github.com/flutter/flutter/issues/148667) , in any case if needed.

### Expected results

The DropdownMenu widget must not be clipped.

### Actual results

The DropdownMenu widget appeares clipped.

### Code sample

<details open><summary>Code sample</summary>

```dart

import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
        useMaterial3: true,
      ),
      home: const MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

const List<String> listArea = <String>[
  "Square Kilometer",
  "Square Meter",
  "Square Mile",
  "Square Feet",
  "Hectare",
  "Acre",
];

class Area {
  final double Ratio__SqFeetToSqKm = 10763910.4;
  final double Ratio__SqMeterToSqKm = 1E6;
  final double Ratio__SqKmToSqMile = 2.59000259000259;
  final double Ratio__HectareToSqKm = 100;
  final double Ratio__AcreToSqKm = 247.1;
  final double Ratio__SqFeetToSqMeter = 10.7639104;
  final double Ratio__SqMeterToSqMile = 2590002.59000259000259;
  final double Ratio__SqMeterToHectare = 10000;
  final double Ratio__AcreToSqMeter = 0.0002471;
  final double Ratio__SqFeetToSqMile = 27878555.81455581455814;
  final double Ratio__HectareToSqMile = 259.00025900025900;
  final double Ratio__AcreToSqMile = 639.98963998963998;
  final double Ratio__SqFeetToHectare = 107639.104;
  final double Ratio__SqFeetToAcre = 43560.94860380412788;
  final double Ratio__AcreToHectare = 2.471;

  Area(); // Zero argument constructor
  Area.squareKilometer(this.squareKilometer) {
    // conversions: (order of assignments is crucial - do not change)
    squareFeet = Ratio__SqFeetToSqKm * squareKilometer;
    squareMeter = squareKilometer * Ratio__SqMeterToSqKm;
    squareMile = squareKilometer / Ratio__SqKmToSqMile;
    hectare = squareKilometer * Ratio__HectareToSqKm;
    acre = squareKilometer * Ratio__AcreToSqKm;
  }
  Area.squareMeter(this.squareMeter) {
    // conversions: (order of assignments is crucial - do not change)
    squareFeet = Ratio__SqFeetToSqMeter * squareMeter;
    squareKilometer = squareMeter / Ratio__SqMeterToSqKm;
    squareMile = squareMeter / Ratio__SqMeterToSqMile;
    hectare = squareMeter / Ratio__SqMeterToHectare;
    acre = squareMeter * Ratio__AcreToSqMeter;
  }
  Area.squareMile(this.squareMile) {
    // conversions:
    squareKilometer = squareMile * Ratio__SqKmToSqMile;
    squareMeter = Ratio__SqMeterToSqMile * squareMile;
    squareFeet = Ratio__SqFeetToSqMile * squareMile;
    hectare = squareMile * Ratio__HectareToSqMile;
    acre = squareMile * Ratio__AcreToSqMile;
  }
  Area.squareFeet(this.squareFeet) {
    // conversions: (order of assignments is crucial - do not change)
    squareMeter = squareFeet / Ratio__SqFeetToSqMeter;
    squareKilometer = squareFeet / Ratio__SqFeetToSqKm;
    squareMile = squareFeet / Ratio__SqFeetToSqMile;
    hectare = squareFeet / Ratio__SqFeetToHectare;
    acre = squareFeet / Ratio__SqFeetToAcre;
  }
  Area.hectare(this.hectare) {
    // conversions: (order of assignments is crucial - do not change)
    squareKilometer = hectare / Ratio__HectareToSqKm;
    squareMeter = hectare * Ratio__SqMeterToHectare;
    squareMile = hectare / Ratio__HectareToSqMile;
    squareFeet = hectare * Ratio__SqFeetToHectare;
    acre = hectare * Ratio__AcreToHectare;
  }
  Area.acre(this.acre) {
    // conversions: (order of assignments is crucial - do not change)
    squareKilometer = acre / Ratio__AcreToSqKm;
    squareMeter = acre / Ratio__AcreToSqMeter;
    squareMile = acre / Ratio__AcreToSqMile;
    squareFeet = acre * Ratio__SqFeetToAcre;
    hectare = acre / Ratio__AcreToHectare;
  }

  double squareKilometer = 0;
  double squareMeter = 0;
  double squareMile = 0;
  double squareFeet = 0;
  double hectare = 0;
  double acre = 0;
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({super.key, required this.title});

  final String title;

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  final GlobalKey _key01 = GlobalKey();
  final GlobalKey _key02 = GlobalKey();

  late Size screenSize;
  double screenWidth = 0;
  double screenHeight = 0;

  final int precisionFlexible = 6;

  double heightAppBar = 0;
  double heightTextField = 0;
  double heightKeyboard = 0;
  double heightOthersCummulative = 15; // include padding heights, etc.
  double heightAdjustment = 60; // additional height adjustment, if needed
  double heightDropdownMenu = 0;

  void setDropdownMenuHeight(
      double heightAppBar, double heightTextField, double heightKeyboard) {
    if (heightAppBar > 0 && heightTextField > 0) {
      heightDropdownMenu = screenHeight -
          heightAppBar -
          heightTextField -
          heightKeyboard -
          heightOthersCummulative -
          heightAdjustment;
    } else {
      heightDropdownMenu = 150;
    }
  }

  late final TextEditingController firstController;
  late final TextEditingController controllerResultSqKilometer;
  late final TextEditingController controllerResultSqMeter;
  late final TextEditingController controllerResultSqMile;
  late final TextEditingController controllerResultSqFeet;
  late final TextEditingController controllerResultHectare;
  late final TextEditingController controllerResultAcre;
  late final TextEditingController controllerResultKhetMuri;
  late final TextEditingController controllerResultRopani;
  late final TextEditingController controllerResultMatoMuri;
  late final TextEditingController controllerResultAana;
  late final TextEditingController controllerResultPaisa;
  late final TextEditingController controllerResultDaam;
  late final TextEditingController controllerResultBigaha;
  late final TextEditingController controllerResultKattha;
  late final TextEditingController controllerResultDhur;

  late String dropdownValue = listArea.first;
  late Area area;

  final FocusNode _focusNode = FocusNode();

  @override
  void initState() {
    super.initState();
    firstController = TextEditingController();

    _focusNode.addListener(_onFocusChange);

    controllerResultSqKilometer = TextEditingController();
    controllerResultSqMeter = TextEditingController();
    controllerResultSqMile = TextEditingController();
    controllerResultSqFeet = TextEditingController();
    controllerResultHectare = TextEditingController();
    controllerResultAcre = TextEditingController();
    controllerResultKhetMuri = TextEditingController();
    controllerResultRopani = TextEditingController();
    controllerResultMatoMuri = TextEditingController();
    controllerResultAana = TextEditingController();
    controllerResultPaisa = TextEditingController();
    controllerResultDaam = TextEditingController();
    controllerResultBigaha = TextEditingController();
    controllerResultKattha = TextEditingController();
    controllerResultDhur = TextEditingController();

    WidgetsBinding.instance.addPostFrameCallback((timeStamp) {
      setState(() {
        heightAppBar = _key01.currentContext?.size?.height ?? -1;
        heightTextField = _key02.currentContext?.size?.height ?? -1;
        heightKeyboard = MediaQuery.of(context).viewInsets.bottom;
        setDropdownMenuHeight(heightAppBar, heightTextField, heightKeyboard);
      });
    });
  }

  void _onFocusChange() {
    Future.delayed(const Duration(milliseconds: 1000), () {
      if (mounted) {
        setState(() {
          heightKeyboard = MediaQuery.of(context).viewInsets.bottom;
          setDropdownMenuHeight(heightAppBar, heightTextField, heightKeyboard);
        });
      }
    });
  }

  @override
  void dispose() {
    firstController.dispose();

    _focusNode.removeListener(_onFocusChange);
    _focusNode.dispose();

    controllerResultSqKilometer.dispose();
    controllerResultSqMeter.dispose();
    controllerResultSqMile.dispose();
    controllerResultSqFeet.dispose();
    controllerResultHectare.dispose();
    controllerResultAcre.dispose();
    controllerResultKhetMuri.dispose();
    controllerResultRopani.dispose();
    controllerResultMatoMuri.dispose();
    controllerResultAana.dispose();
    controllerResultPaisa.dispose();
    controllerResultDaam.dispose();
    controllerResultBigaha.dispose();
    controllerResultKattha.dispose();
    controllerResultDhur.dispose();

    super.dispose();
  }

  void changeResult(String? dropdownValue) {
    this.dropdownValue = dropdownValue!;

    if (firstController.text.isNotEmpty) {
      switch (dropdownValue) {
        case "Square Kilometer":
          area = Area.squareKilometer(double.parse(firstController.text));
          controllerResultSqKilometer.text = area.squareKilometer.toString();
          controllerResultSqMeter.text = area.squareMeter.toString();
          controllerResultSqMile.text = area.squareMile.toString();
          controllerResultSqFeet.text = area.squareFeet.toString();
          controllerResultHectare.text = area.hectare.toString();
          controllerResultAcre.text = area.acre.toString();
          break;
        case "Square Meter":
          area = Area.squareMeter(double.parse(firstController.text));
          controllerResultSqKilometer.text = area.squareKilometer.toString();
          controllerResultSqMeter.text = area.squareMeter.toString();
          controllerResultSqMile.text = area.squareMile.toString();
          controllerResultSqFeet.text = area.squareFeet.toString();
          controllerResultHectare.text = area.hectare.toString();
          controllerResultAcre.text = area.acre.toString();
          break;
        case "Square Mile":
          area = Area.squareMile(double.parse(firstController.text));
          controllerResultSqKilometer.text = area.squareKilometer.toString();
          controllerResultSqMeter.text = area.squareMeter.toString();
          controllerResultSqMile.text = area.squareMile.toString();
          controllerResultSqFeet.text = area.squareFeet.toString();
          controllerResultHectare.text = area.hectare.toString();
          controllerResultAcre.text = area.acre.toString();
          break;
        case "Square Feet":
          area = Area.squareFeet(double.parse(firstController.text));
          controllerResultSqKilometer.text = area.squareKilometer.toString();
          controllerResultSqMeter.text = area.squareMeter.toString();
          controllerResultSqMile.text = area.squareMile.toString();
          controllerResultSqFeet.text = area.squareFeet.toString();
          controllerResultHectare.text = area.hectare.toString();
          controllerResultAcre.text = area.acre.toString();
          break;
        case "Hectare":
          area = Area.hectare(double.parse(firstController.text));
          controllerResultSqKilometer.text = area.squareKilometer.toString();
          controllerResultSqMeter.text = area.squareMeter.toString();
          controllerResultSqMile.text = area.squareMile.toString();
          controllerResultSqFeet.text = area.squareFeet.toString();
          controllerResultHectare.text = area.hectare.toString();
          controllerResultAcre.text = area.acre.toString();
          break;
        case "Acre":
          area = Area.acre(double.parse(firstController.text));
          controllerResultSqKilometer.text = area.squareKilometer.toString();
          controllerResultSqMeter.text = area.squareMeter.toString();
          controllerResultSqMile.text = area.squareMile.toString();
          controllerResultSqFeet.text = area.squareFeet.toString();
          controllerResultHectare.text = area.hectare.toString();
          controllerResultAcre.text = area.acre.toString();
          break;
      }
    } else {
      controllerResultSqKilometer.text = "";
      controllerResultSqMeter.text = "";
      controllerResultSqMile.text = "";
      controllerResultSqFeet.text = "";
      controllerResultHectare.text = "";
      controllerResultAcre.text = "";
    }
  }

  @override
  Widget build(BuildContext context) {
    screenSize = MediaQuery.of(context).size;
    screenWidth = screenSize.width;
    screenHeight = screenSize.height;

    return Scaffold(
      appBar: AppBar(
        key: _key01,
        backgroundColor: Theme.of(context).colorScheme.inversePrimary,
        title: Text(widget.title),
      ),
      body: GestureDetector(
        onTap: () {
          FocusScope.of(context).requestFocus(FocusNode());
        },
        child: Container(
          padding: const EdgeInsets.all(5.0),
          child: Column(
            children: [
              Container(
                decoration: const BoxDecoration(
                  color: Colors.transparent,
                  borderRadius: BorderRadius.all(
                    Radius.circular(5.0),
                  ),
                ),
                child: Row(
                  children: [
                    Flexible(
                      flex: 1,
                      child: Container(
                        padding: const EdgeInsets.all(5.0),
                        child: TextField(
                          key: _key02,
                          style: const TextStyle(
                            fontSize: 16.0,
                          ),
                          controller: firstController,
                          decoration: const InputDecoration(
                            border: OutlineInputBorder(),
                            // hintText: 'Value',
                            labelText: 'Value',
                          ),
                          keyboardType: TextInputType.number,
                          focusNode: _focusNode,
                          onChanged: (value) {
                            changeResult(dropdownValue);
                          },
                        ),
                      ),
                    ),
                    Flexible(
                      flex: 1,
                      child: Container(
                        padding: const EdgeInsets.all(5.0),
                        child: DropdownMenu<String>(
                          menuHeight: heightDropdownMenu,
                          textStyle: const TextStyle(
                            fontSize: 16.0,
                          ),
                          initialSelection: dropdownValue,
                          onSelected: (String? value) {
                            setState(() {
                              changeResult(value);
                            });
                          },
                          dropdownMenuEntries: listArea
                              .map<DropdownMenuEntry<String>>((String value) {
                            return DropdownMenuEntry<String>(
                                value: value, label: value);
                          }).toList(),
                        ),
                      ),
                    ),
                  ],
                ),
              ),
              Expanded(
                child: SingleChildScrollView(
                  child: Column(
                    children: [
                      // Square Kilometer
                      Container(
                        decoration: const BoxDecoration(
                          color: Color.fromRGBO(245, 245, 245, 1.0),
                          borderRadius: BorderRadius.all(
                            Radius.circular(5.0),
                          ),
                        ),
                        child: Row(
                          children: [
                            Expanded(
                              flex: 5,
                              child: Container(
                                padding: const EdgeInsets.all(5.0),
                                child: TextField(
                                  readOnly: true,
                                  style: const TextStyle(
                                    fontSize: 16.0,
                                  ),
                                  decoration: const InputDecoration(
                                      border: InputBorder.none),
                                  controller: controllerResultSqKilometer,
                                ),
                              ),
                            ),
                            Expanded(
                              flex: 5,
                              child: Container(
                                  padding: const EdgeInsets.all(5.0),
                                  child: const Text(
                                    "Square Kilometer",
                                    style: TextStyle(
                                      fontSize: 16.0,
                                    ),
                                  )),
                            ),
                          ],
                        ),
                      ),
                      // Square Meter
                      Container(
                        decoration: const BoxDecoration(
                          color: Colors.transparent,
                          borderRadius: BorderRadius.all(
                            Radius.circular(5.0),
                          ),
                        ),
                        child: Row(
                          children: [
                            Expanded(
                              flex: 5,
                              child: Container(
                                padding: const EdgeInsets.all(5.0),
                                child: TextField(
                                  readOnly: true,
                                  style: const TextStyle(
                                    fontSize: 16.0,
                                  ),
                                  decoration: const InputDecoration(
                                      border: InputBorder.none),
                                  controller: controllerResultSqMeter,
                                ),
                              ),
                            ),
                            Expanded(
                              flex: 5,
                              child: Container(
                                  padding: const EdgeInsets.all(5.0),
                                  child: const Text(
                                    "Square Meter",
                                    style: TextStyle(
                                      fontSize: 16.0,
                                    ),
                                  )),
                            ),
                          ],
                        ),
                      ),
                      // Square Mile
                      Container(
                        decoration: const BoxDecoration(
                          color: Color.fromRGBO(245, 245, 245, 1.0),
                          borderRadius: BorderRadius.all(
                            Radius.circular(5.0),
                          ),
                        ),
                        child: Row(
                          children: [
                            Expanded(
                              flex: 5,
                              child: Container(
                                padding: const EdgeInsets.all(5.0),
                                child: TextField(
                                  readOnly: true,
                                  style: const TextStyle(
                                    fontSize: 16.0,
                                  ),
                                  decoration: const InputDecoration(
                                      border: InputBorder.none),
                                  controller: controllerResultSqMile,
                                ),
                              ),
                            ),
                            Expanded(
                              flex: 5,
                              child: Container(
                                  padding: const EdgeInsets.all(5.0),
                                  child: const Text(
                                    "Square Mile",
                                    style: TextStyle(
                                      fontSize: 16.0,
                                    ),
                                  )),
                            ),
                          ],
                        ),
                      ),
                      // Square Feet
                      Container(
                        decoration: const BoxDecoration(
                          color: Colors.transparent,
                          borderRadius: BorderRadius.all(
                            Radius.circular(5.0),
                          ),
                        ),
                        child: Row(
                          children: [
                            Expanded(
                              flex: 5,
                              child: Container(
                                padding: const EdgeInsets.all(5.0),
                                child: TextField(
                                  readOnly: true,
                                  style: const TextStyle(
                                    fontSize: 16.0,
                                  ),
                                  decoration: const InputDecoration(
                                      border: InputBorder.none),
                                  controller: controllerResultSqFeet,
                                ),
                              ),
                            ),
                            Expanded(
                              flex: 5,
                              child: Container(
                                  padding: const EdgeInsets.all(5.0),
                                  child: const Text(
                                    "Square Feet",
                                    style: TextStyle(
                                      fontSize: 16.0,
                                    ),
                                  )),
                            ),
                          ],
                        ),
                      ),
                      // Hectare
                      Container(
                        decoration: const BoxDecoration(
                          color: Color.fromRGBO(245, 245, 245, 1.0),
                          borderRadius: BorderRadius.all(
                            Radius.circular(5.0),
                          ),
                        ),
                        child: Row(
                          children: [
                            Expanded(
                              flex: 5,
                              child: Container(
                                padding: const EdgeInsets.all(5.0),
                                child: TextField(
                                  readOnly: true,
                                  style: const TextStyle(
                                    fontSize: 16.0,
                                  ),
                                  decoration: const InputDecoration(
                                      border: InputBorder.none),
                                  controller: controllerResultHectare,
                                ),
                              ),
                            ),
                            Expanded(
                              flex: 5,
                              child: Container(
                                  padding: const EdgeInsets.all(5.0),
                                  child: const Text(
                                    "Hectare",
                                    style: TextStyle(
                                      fontSize: 16.0,
                                    ),
                                  )),
                            ),
                          ],
                        ),
                      ),
                      // Acre
                      Container(
                        decoration: const BoxDecoration(
                          color: Colors.transparent,
                          borderRadius: BorderRadius.all(
                            Radius.circular(5.0),
                          ),
                        ),
                        child: Row(
                          children: [
                            Expanded(
                              flex: 5,
                              child: Container(
                                padding: const EdgeInsets.all(5.0),
                                child: TextField(
                                  readOnly: true,
                                  style: const TextStyle(
                                    fontSize: 16.0,
                                  ),
                                  decoration: const InputDecoration(
                                      border: InputBorder.none),
                                  controller: controllerResultAcre,
                                ),
                              ),
                            ),
                            Expanded(
                              flex: 5,
                              child: Container(
                                  padding: const EdgeInsets.all(5.0),
                                  child: const Text(
                                    "Acre",
                                    style: TextStyle(
                                      fontSize: 16.0,
                                    ),
                                  )),
                            ),
                          ],
                        ),
                      ),
                    ],
                  ),
                ),
              ),
            ],
          ),
        ),
      ),
    );
  }
}


```

</details>

### Screenshots or Video

<details open>
<summary>Screenshots / Video demonstration</summary>

![Screenshot_1717550074](https://github.com/flutter/flutter/assets/170310374/7dc8df4e-8d90-4a9a-9f0c-fe95a0b511ae)

</details>

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```console

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.22.0, on Ubuntu 22.04.4 LTS 6.5.0-35-generic, locale en_GB.UTF-8)
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✓] Chrome - develop for the web
[✓] Linux toolchain - develop for Linux desktop
[✓] Android Studio (version 2023.2)
[✓] VS Code (version 1.89.1)
[✓] Connected device (3 available)
[✓] Network resources

• No issues found!

```

</details>

**Comment by huycozy** [comment]
I can reproduce this issue with much more minimal code as below on Flutter stable 3.22.1. But it seems to be fixed on Flutter master channel as I don't see the issue anymore. 

<details>
<summary>Minimal sample code</summary>

```dart

import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  @override
  Widget build(BuildContext context) {
    return const MaterialApp(
      title: 'Flutter Demo',
      home: MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

const List<String> listArea = <String>[
  "Square Kilometer",
  "Square Meter",
  "Square Mile",
  "Square Feet",
  "Hectare",
  "Acre",
];

class MyHomePage extends StatefulWidget {
  const MyHomePage({super.key, required this.title});

  final String title;

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        backgroundColor: Theme.of(context).colorScheme.inversePrimary,
        title: Text(widget.title),
      ),
      body: Row(
        children: [
          Flexible(
            flex: 1,
            child: Container(
              padding: const EdgeInsets.all(5.0),
              child: TextField(
                style: const TextStyle(
                  fontSize: 16.0,
                ),
                decoration: const InputDecoration(
                  border: OutlineInputBorder(),
                  labelText: 'Value',
                ),
                keyboardType: TextInputType.number,
                onChanged: (value) {
                },
              ),
            ),
          ),
          Flexible(
            child: Container(
              padding: const EdgeInsets.all(5.0),
              child: DropdownMenu<String>(
                initialSelection: listArea[0],
                dropdownMenuEntries: listArea
                    .map<DropdownMenuEntry<String>>((String value) {
                  return DropdownMenuEntry<String>(
                      value: value, label: value);
                }).toList(),
              ),
            ),
          ),
        ],
      ),
    );
  }
}


```
</details>

<details open>
<summary>Demo fixed on Flutter master channel</summary>

| more minimal code | Original Poster code |
| --------------- | --------------- |
<img width="400" src="https://github.com/flutter/flutter/assets/104349824/8c673c5a-f12d-4654-8413-6b24d3004fa8"> | <img width="400" src="https://github.com/flutter/flutter/assets/104349824/79f2825f-3274-4d05-91c5-4d2365ee0862">
</details>

<details>
<summary>flutter doctor -v (stable and master)</summary>

```bash
[✓] Flutter (Channel stable, 3.22.1, on macOS 14.1 23B74 darwin-x64, locale en-VN)
    • Flutter version 3.22.1 on channel stable at /Users/huynq/Documents/GitHub/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision a14f74ff3a (13 days ago), 2024-05-22 11:08:21 -0500
    • Engine revision 55eae6864b
    • Dart version 3.4.1
    • DevTools version 2.34.3

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /Users/huynq/Library/Android/sdk
    • Platform android-34, build-tools 34.0.0
    • ANDROID_HOME = /Users/huynq/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.10+0-17.0.10b1087.21-11572160)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 15.3)
    • Xcode at /Applications/Xcode15.3.app/Contents/Developer
    • Build 15E204a
    • CocoaPods version 1.15.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2023.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • android-studio-dir = /Applications/Android Studio.app/
    • Java version OpenJDK Runtime Environment (build 17.0.10+0-17.0.10b1087.21-11572160)

[✓] VS Code (version 1.89.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.90.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • macOS 14.1 23B74 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 125.0.6422.142

[✓] Network resources
    • All expected network resources are available.

• No issues found!
```

```bash
[!] Flutter (Channel master, 3.23.0-13.0.pre.100, on macOS 14.1 23B74 darwin-x64, locale en-VN)
    • Flutter version 3.23.0-13.0.pre.100 on channel master at /Users/huynq/Documents/GitHub/flutter_master
    ! Warning: `flutter` on your path resolves to /Users/huynq/Documents/GitHub/flutter/bin/flutter, which is not inside your current Flutter SDK checkout at /Users/huynq/Documents/GitHub/flutter_master. Consider adding /Users/huynq/Documents/GitHub/flutter_master/bin to the front of your path.
    ! Warning: `dart` on your path resolves to /Users/huynq/Documents/GitHub/flutter/bin/dart, which is not inside your current Flutter SDK checkout at /Users/huynq/Documents/GitHub/flutter_master. Consider adding /Users/huynq/Documents/GitHub/flutter_master/bin to the front of your path.
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 63daac2a51 (2 hours ago), 2024-06-04 21:16:19 -0400
    • Engine revision 11a32d43e3
    • Dart version 3.5.0 (build 3.5.0-222.0.dev)
    • DevTools version 2.36.0
    • If those were intentional, you can disregard the above warnings; however it is recommended to use "git" directly to perform update checks and upgrades.

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /Users/huynq/Library/Android/sdk
    • Platform android-34, build-tools 34.0.0
    • ANDROID_HOME = /Users/huynq/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.10+0-17.0.10b1087.21-11572160)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 15.3)
    • Xcode at /Applications/Xcode15.3.app/Contents/Developer
    • Build 15E204a
    • CocoaPods version 1.15.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2023.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • android-studio-dir = /Applications/Android Studio.app/
    • Java version OpenJDK Runtime Environment (build 17.0.10+0-17.0.10b1087.21-11572160)

[✓] VS Code (version 1.89.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.90.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • macOS 14.1 23B74 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 125.0.6422.142

[✓] Network resources
    • All expected network resources are available.

! Doctor found issues in 1 category.
```
</details>

So please switch to Flutter master channel and confirm.

**Comment by huycozy** [comment]
The fix should land on `beta` channel first then `stable` afterwards. So you can wait and use `beta` channel as it's more stable than `master` channel. You can read the details here: https://github.com/flutter/flutter/blob/master/docs/releases/Flutter-build-release-channels.md.

Another option in case you can't wait is: you do `git bisect` to find a fixed/good commit between `stable` and `master` channels. Then pick that commit to your local Flutter SDK (on stable channel).

Since this is fixed on master channel, I will close the issue. Thanks for confirming this.

**Comment by aj-Thapa** [comment]
Subject: Request for deletion of this issue (#149706) page.

I'm writing this request to get this Flutter issue (#149706) page delete. I've published full working code in the page that I'm looking to protect from others copying them. I had to publish full functional code as the minimal code to reproduce the problem; in lesser code, the problem would not appear. But now that the issue closed here and solved in the Master channel of Flutter, I request you to delete this issue page. Thank you.
